Package.swift

```diff
@@ -18,6 +18,15 @@
  **/
 
 import PackageDescription
+import Foundation
+
+var kituraNetPackage: Package.Dependency
+
+if let _ = ProcessInfo.processInfo.environment["KITURA_NIO"] {
+    kituraNetPackage = .package(url: "https://github.com/IBM-Swift/Kitura-NIO.git", from: "1.0.0")
```

Comment:
I wonder if there is a case for tagging Kitura-NIO to 2.0.0, just to keep things closer in sync?

Reply:
any comments on that @djones6 ?